kotlin/src/main/kotlin/org/axonframework/extensions/kotlin/QueryGatewayExtensions.kt

```diff
@@ -25,29 +25,22 @@ import java.util.*
 import java.util.concurrent.CompletableFuture
 
```

Comment:
So, we do not need extension functions for `scatterGather` and `subscriptionQuery` just yet?

Reply:
Those would be useful, created https://github.com/AxonFramework/extension-kotlin/issues/16 and  https://github.com/AxonFramework/extension-kotlin/issues/17 to implement them